CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+<!-- SPDX-License-Identifier: CC0-1.0 -->
+# Contributing to Govdirectory
+
+🙇‍♀️ Thank you for contributing!
+
+We love issues and pull requests from everyone.
+If you're not comfortable with those features of GitHub, you can also just [start a discussion](https://github.com/govdirectory/website/discussions).
+
+## Problems, suggestions and questions in issues
+
+Please help development by reporting problems, suggesting changes and asking questions.
+To do this, you can [create a GitHub issue](https://help.github.com/articles/creating-an-issue/) for this project in the [GitHub Issues for the website](https://github.com/govdirectory/website/issues).
+
+You don't need to change any of our code or documentation to be a contributor!
+
+## Documentation and code in pull requests
+
+If you want to add to the documentation or code of one of our projects you should make a pull request.
+
+If you never used GitHub, get up to speed with [Understanding the GitHub flow](https://guides.github.com/introduction/flow/) or follow one of the great free interactive courses in the [GitHub learning lab](https://lab.github.com/) on working with GitHub and working with MarkDown, the syntax this project's documentation is in.
+
+This project is [licensed CC-0](LICENSE.md), which essentially means that the project, along with your contributions is in the public domain in whatever jurisdiction possible, and everyone can do whatever they want with it.
+
+### 1. Make your changes
+
+This project uses the [GitFlow branching model and workflow](https://nvie.com/posts/a-successful-git-branching-model/).
+When you've forked this repository, please make sure to create a feature branch following the GitFlow model.
+
+Add your changes in commits [with a message that explains them](https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message).
+Document choices or decisions you make in the commit message, this will enable everyone to be informed of your choices in the future.
+
+If you are adding code, make sure you've added and updated the relevant documentation and tests before you submit your pull request.
+Make sure to write tests that show the behavior of the newly added or changed code.
+
+### 2. Pull request
+
+When submitting the pull request, please accompany it with a description of the problem you are trying to solve and the issue numbers that this pull request fixes.
+
+### 3. Improve
+
+All contributions have to be reviewed by someone.
+
+It could be that your contribution can be merged immediately by a maintainer.
```

Comment:
Markdown ignores these newlines, do we want to turn it into a list or remove the newlines and make it a paragraph?

Reply:
I have specifically used newlines that just renders as a space since future changes will provide much cleaner diffs (that is, it will enable a diff per sentence instead per paragraph).